File: Assets/Scripts/GameManager.js
#pragma strict

var creep : GameObject;
var spawn_position;
var timer = 0.0;

function spawnCreep(){
spawn_position = Vector3(0,0,0);
var temp_spawn = Instantiate(creep,spawn_position, Quaternion.identity);
}
function Start () {

}

function Update () {
	timer+= Time.deltaTime;
	if(timer>15){
		spawnCreep();
		timer = 0.0;
	}
}

File: Assets/Scripts/GameManager.cs
using UnityEngine;
using System.Collections;

public class GameManager : MonoBehaviour {
	public GameObject creep;
	public Vector2 spawnValues;
	public int creepCount;
	void Start (){
		SpawnCreeps ();
	}
	void SpawnCreeps(){
		for (int i=0;i<creepCount;i++) {
			Vector2 spawnPosition = new Vector2 (5, 5);
			Quaternion spawnRotation = new Quaternion ();
			Instantiate (creep, spawnPosition, spawnRotation);
		}
	}
}
